Documentation/deploy-master.md

```diff
@@ -154,6 +160,7 @@ spec:
     - --tls-private-key-file=/etc/kubernetes/ssl/apiserver-key.pem
     - --client-ca-file=/etc/kubernetes/ssl/ca.pem
     - --service-account-key-file=/etc/kubernetes/ssl/apiserver-key.pem
+    - --runtime-config=extensions/v1beta1=true,extensions/v1beta1/thirdpartyresources=true
```

Comment:
I'm fine with not explaining for now, but we can open up a side-issue if we want to add some docs. We've never had a question about it, short of people asking us to enable them.